docs: add link to Bridge In Tech zulip stream in CONTRIBUTING.md (#219)
anitab-org/bridge-in-tech-web: .github/CONTRIBUTING.md

# Contributing Guidelines

> * You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). You can find the Bridge in Tech stream [here](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech).

Reviewer: ```suggestion
* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Bridge in Tech stream is [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech).
```

Author: Will make these changes :)
